packages/logger/src/winston-logger/logger.ts

```diff
@@ -39,6 +41,18 @@ export class WinstonLogger implements Logger {
     debug(message: string | object): void {
         this.transportMap.size && this._logger.debug(message);
     }
+    log(data: string | Log): void {
+        if (!this.transportMap.size) {
+            // Nothing to do
+            return;
+        }
+        if (typeof data === 'string') {
+            this._logger.log(this.winstonLevel, data);
+        } else {
+            this._logger.log(toWinstonLogLevel(data.level)!, data.message);
```

Comment:
Is the `!` on purpose?

Reply:
Yes, it was on purpose but it's gone now.